spec/system/service_unavailable_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'Service Unavailable' do
+  before do
+    allow(ENV).to receive(:[]).and_call_original
+    allow(ENV).to receive(:[]).with('MAINTENANCE').and_return('true')
+  end
+
+  context 'when the service is unavailable and user navigates to home' do
+    before do
+      visit '/service-unavailable'
+      get '/'
+    end
+
+    it 'redirects to the service unavailable page' do
+      expect(response).to redirect_to '/service-unavailable'
+    end
+
+    it 'the page has the correct content' do
+      expect(page).to have_css('h1', text: 'Sorry, the service is unavailable')
+      expect(page).to have_title('Early years child development training : Sorry, the service is unavailable')
+    end
+  end
+
+  context 'when the service is unavailable and user navigates to my modules page' do
+    before do
+      get my_modules_path
```

Comment:
Because this is a system spec and not a request spec I'd suggest using `visit` throughout.

Reply:
[37aa9b6](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/941/commits/37aa9b6ee52227bd8b6d9dd34c555eab44b37802)